ONSdigital/babbage
# Fix - Add download descriptions to download as links

### What
Make the download as links more understandable to screen reader users.

This affects
1. Charts,
1. Tables,
1. Print options (top in bulletins)
1. Dataset list pages
1. Timeseries pages

Additionally - fix the whitespace formatting on timeseries page (you can ignore the whitespace changes in diff options on github)

### How to review
1. Load a page
1. See that the download links are read just as `xls`, `image` etc
1. Switch to this branch 
1. See they are read correctly

### Who can review
Anyone but me
